CHANGELOG.md

```diff
@@ -42,7 +44,7 @@ functionality to [lifecycles](https://developer.hashicorp.com/terraform/language
 
 - Move archive pipeline config to provider [[PR #354](https://github.com/buildkite/terraform-provider-buildkite/pull/354)] @jradtilbrook
 - SUP-1076 Convert testing to framework [[PR #361](https://github.com/buildkite/terraform-provider-buildkite/pull/361)] @jradtilbrook
-    - SUP-1076 Move Cluster tests to use t.Run [[PR #365](https://github.com/buildkite/terraform-provider-buildkite/pull/365)] @mcncl
+- SUP-1076 Move Cluster tests to use t.Run [[PR #365](https://github.com/buildkite/terraform-provider-buildkite/pull/365)] @mcncl
```

Comment:
This indenting is intentional; it puts the Cluster test changes as a part of the above bullet point.

Reply:
Ah nice - Ive rolled it back with the latest